test/libsolidity/SemanticTest.cpp

```diff
@@ -236,8 +249,23 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 			return TestResult::Failure;
 		}
 
+		if (success && !m_runWithEwasm && _compileToEwasm)
+		{
+			if (m_revertStrings != RevertStrings::Default)
+				return TestResult::Success;
+
+			m_compileToEwasmCanBeSet = true;
+			AnsiColorized(_stream, _formatted, {BOLD, YELLOW}) <<
+				_linePrefix << endl <<
+				_linePrefix << "Test can pass via Yul (ewasm) and marked with compileToEwasm: false." << endl;
```

Comment:
Sorry for being pedantic @aarlt , but I still don't understand this sentence. Why would be marked with `compileToEwasm: false` if it `passes via Yul (ewasm)`?

Reply:
Ah, yes, true. It's a bit confusing.. will change that into something like `Test can pass via Yul (ewasm), but was marked with compileToEwasm: false.`.